src/pkg/packaging/deb/package.targets

```diff
@@ -165,38 +223,74 @@
     <MakeDir Directories="$(debLayoutPackageRoot)" />
     <MakeDir Directories="$(debLayoutSamples)" />
     <MakeDir Directories="$(debLayoutDocs)" />
-    
+
     <!-- Copy files to debian layout -->
     <ItemGroup>
-        <SHFiles Include="$(InputRoot)/**/*" />
-        <SHManpages Include="$(ManPagesDir)/**/*" />
+      <SHFiles Include="$(InputRoot)/**/*" />
```

Comment:
Are there any cases where InputRoot or ManPagesDir is empty? If so then this globbing will cause you to craw your entire harddrive and kill build performance.